pages/open-positions.md

```diff
@@ -5,6 +5,15 @@ redirect_from: /roles-and-teams/
 styles: assets/css/leverpostings.css
 scripts: assets/js/leverpostings.js
 ---
+
+###Important Announcement###
+On May 1, we’ll be taking down our open positions for a short application intake hiatus. During that time, we’ll be evolving how 18F plans for open roles. We estimate that we’ll post new open positions in the beginning of June and suggest that you check back here at that time.
```

Comment:
@fureigh Since the newsletter is TBD, I think the only way they will know about the new position is checking the site. Thought that we should just say that?